internal/driver/glfw/window.go

```diff
@@ -511,7 +511,7 @@ func (w *window) processMouseClicked(button desktop.MouseButton, action action,
 
 	co, pos, _ := w.findObjectAtPositionMatching(w.canvas, mousePos, func(object fyne.CanvasObject) bool {
 		switch object.(type) {
-		case fyne.Tappable, fyne.SecondaryTappable, fyne.DoubleTappable, fyne.Focusable, desktop.Mouseable:
+		case fyne.Tappable, fyne.SecondaryTappable, fyne.DoubleTappable, fyne.Focusable, desktop.Mouseable, desktop.Hoverable:
```

Comment:
I don't agree with this change.

If you put a hoverable item over a tappable item then the event will no longer get sent to the tappable item because It will be swallowed by this check for coverable.